app/src/features/settings/components/BillingTeam/components/RequestTeamAccessModal/index.tsx

```diff
@@ -9,30 +11,79 @@ import APP_CONSTANTS from "config/constants";
 import "./index.scss";
 
 interface Props {
-  isOpen: boolean;
-  onClose: () => void;
+  isOpen?: boolean;
+  onClose?: () => void;
+  isReminder?: boolean;
 }
 
-export const RequestBillingTeamAccessModal: React.FC<Props> = ({ isOpen, onClose }) => {
+export const RequestBillingTeamAccessModal: React.FC<Props> = ({ isOpen, onClose, isReminder = false }) => {
+  const joinTeamReminder = useFeatureValue("join_team_reminder", null);
+  const user = useSelector(getUserAuthDetails);
   const billingTeams = useSelector(getAvailableBillingTeams);
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [isModalClosable, setIsModalClosable] = useState(true);
+
+  const emailSubject = "Request a new Billing Team";
+  const body = `Hey Requestly Team\n\n We'd like to setup a new Billing Team. Could you please assist with the next step here?\n\nThanks\n${user?.details?.profile?.displayName}`;
 
   useEffect(() => {
-    if (isOpen) {
+    if (isOpen || isModalVisible) {
       trackRequestBillingTeamAccessModalViewed();
     }
   }, [isOpen]);
 
+  useEffect(() => {
+    if (joinTeamReminder && isReminder) {
```

Comment:
this is the main useEffect that makes me believe that all these changes should be a separate component since this is specific to the reminder case